.github/workflows/t3000-unit-tests-impl.yaml

```diff
@@ -13,7 +13,7 @@ jobs:
           { name: "t3k ttnn tests", arch: wormhole_b0, cmd: run_t3000_ttnn_tests, timeout: 120, owner_id: UBHPP2NDP}, #Joseph Chu
           { name: "t3k falcon7b tests", arch: wormhole_b0, cmd: run_t3000_falcon7b_tests, timeout: 30, owner_id: UBHPP2NDP}, #Joseph Chu
           { name: "t3k falcon40b tests", arch: wormhole_b0, cmd: run_t3000_falcon40b_tests, timeout: 30, owner_id: U053W15B6JF}, #Djordje Ivanovic
-          { name: "t3k llama3 tests", arch: wormhole_b0, cmd: run_t3000_llama3_tests, timeout: 30, owner_id: U03PUAKE719},  #Miguel Tairum Cruz
+          { name: "t3k llama3 tests", arch: wormhole_b0, cmd: run_t3000_llama3_tests, timeout: 60, owner_id: U03PUAKE719},  #Miguel Tairum Cruz
```

Comment:
note that these are unit tests and meant to be quick tests. is it possible to not extend the timeout to 60 minutes?

Reply:
Yes, I think that 45 is probably enough.

I understand these are unit tests, but the new Llama3 T3K tests is not a single model, but in fact 4 models: 
- Llama3.1-8B
- Llama3.2-1B
- Llama3.2-3B
- Llama3.2-11B

All same codebase (exact same test for every model above) but different params which requires us to actually test them.

Since we added the 11B with this PR, 30 min is not enough to run the 4 models. In reality the timeout it's actually tighter than other models: 1h for 4 models, as opposed to ~30 min per model.

Reply:
Let me know if this is good @ttmchiou. 
If not I can adjust the timeout down a bit for now, but keep in mind that we are in the process of adding Llama3-70B so it's likely that we'll have to raise the timeout again.

Reply:
Alternatively, if more and more Llama3 models are added we can definitely create a separate job for them:

- Something like:
- llama3-small (including 1B, 3B and 8B)
- llama3-large (including 11B and 70B and maybe 90B)